config/software/stunnel.rb

```diff
@@ -34,7 +42,7 @@
 build do
   env = with_standard_compiler_flags(with_embedded_path)
   configure_string = <<EOF
-./configure --with-ssl=#{install_dir}/embedded --prefix=#{install_dir}/embedded
+./configure --with-ssl=#{install_dir}/embedded --prefix=#{install_dir}/embedded --enable-fips
```

Comment:
Is it worth doing a conditional similar to https://github.com/chef/omnibus-software/blob/master/config/software/openssl.rb#L76-L78 ?